_security/multi-tenancy/tenant-index.md

```diff
@@ -10,18 +10,19 @@ redirect_from:
 
 # OpenSearch Dashboards multi-tenancy
 
-*Tenants* in OpenSearch Dashboards are spaces for saving index patterns, visualizations, dashboards, and other OpenSearch Dashboards objects. Tenants are useful for safely sharing your work with other OpenSearch Dashboards users. You can control which roles have access to a tenant and whether those roles have read or write access. By default, all OpenSearch Dashboards users have access to two independent tenants:
+*Tenants* in OpenSearch Dashboards are spaces for saving index patterns, visualizations, dashboards, and other OpenSearch Dashboards objects. OpenSearch allows users to create multiple tenants for multiple uses. Tenants are useful for safely sharing your work with other OpenSearch Dashboards users. You can control which roles have access to a tenant and whether those roles have read or write access. By default, all OpenSearch Dashboards users have access to two independent tenants: private and global. Multi-tenancy also provides the option to create custom tenants.
```

Comment:
Same comment re: "private" and "global".

Reply:
I don't so in this case. First, I don't think the global tenant needs to be a proper noun. Where it's referred to as a setting, and it's capitalized in the UI or a key, sure. Likewise with private tenants. Every user gets one. So there's no monolithic private tenant. Just a bunch of separate ones. 
Still, I've reworded this like so:
"... all OpenSearch Dashboards users have access to two independent tenants: the global tenant and a private tenant."